system/settings/settings.xml

```diff
@@ -1008,18 +1008,21 @@
           <default>false</default>
           <control type="toggle" />
         </setting>
-        <setting id="videolibrary.groupsingleitemsets" type="boolean" label="20470" help="36157">
+        <setting id="videolibrary.groupsingleitemsets" type="boolean" parent="videolibrary.groupmoviesets" label="20470" help="36157">
           <level>1</level>
           <default>false</default>
           <control type="toggle" />
+          <dependencies>
+            <dependency type="enable" setting="videolibrary.groupmoviesets" operator="is">true</dependency>
+          </dependencies>
         </setting>
```

Comment:
Hadn't noticed before that ""videolibrary.groupsingleitemsets" applied to sets node as well, so will remove the dependancy, so question is does this

![image](https://cloud.githubusercontent.com/assets/5781142/16186588/3e11a952-36c3-11e6-942e-14ef7d9b8583.png)

or this

![image](https://cloud.githubusercontent.com/assets/5781142/16186636/86810fd4-36c3-11e6-960e-491552046620.png)

make more sense.